apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -744,11 +744,22 @@ export class InstallManager {
             noMtime: true,
             noPax: true,
             sync: true,
-            prefix: npmPackageFolder
+            prefix: npmPackageFolder,
+            filter: (path: string, stat: tar.FileStat): boolean => {
+              if (!this._rushConfiguration.experimentsConfiguration.configuration
+                .noChmodFieldInTarHeaderNormalization) {
+
+                stat.mode = (stat.mode & ~0x1FF) | PosixModeBits.AllRead | PosixModeBits.UserWrite
+                  | PosixModeBits.AllExecute;
+              }
+
+              return true;
+            }
           } as CreateOptions, [FileConstants.PackageJson]);
```

Comment:
Can we remove this cast?

Reply:
I don't think we can.

```
/**
 * The "noMtime" flag is new in tar@4.4.1 and not available yet for \@types/tar.
 * As a temporary workaround, augment the type.
 */
import { CreateOptions } from 'tar';

```

Reply:
Or just open a PR against `@types/tar`. Often such PRs get merged+published pretty fast.